Week2/homework/js-exercises/gradeCalc.js

```diff
@@ -1,28 +1,57 @@
 "use strict";
-function vathmos(score){
-    let grade;
-    switch(true) {
-        case score < 50: 
-          grade = " You got an F " + parseFloat(score)+"%";
-          break;
-        case score < 60: 
-          grade = " You got a E " + parseFloat(score)+"%";
-          break;
-        case score < 70: 
-          grade =" You got a D " + parseFloat(score)+"%";
-          break;
-        case score < 80: 
-          grade = " You got a C"  + parseFloat(score)+"%";
-          break;
-          case score < 90: 
-          grade = "You got a B " + parseFloat(score)+"%";
-          break;
-        case score <= 100:
-          grade = "You got a A " + parseFloat(score)+"%";
-          break;
-      }
-      return grade;
+let gradeCalc = function(score, totalScore){
+  let percent = (score / totalScore) *100;
+  let letterGrade = "";
+      if(percent >= 90){
+          letterGrade = "A";     
+         } else if(percent >= 80){
+          letterGrade = "B";
+         }else if (percent >=70){
+             letterGrade = "C";
+         }else if(percent >=60){
+             letterGrade = "D";
+         }else {
+             letterGrade = "E";
+         }
+
+    return `You got a ${letterGrade} (${percent}%)!`
+
 }
+let result = gradeCalc(13,20);
+console.log(result);
+
+
+
+
+
+
+
+
+
+// function vathmos(score){
+//     let grade;
+//     switch(true) {
+//         case score < 50: 
+//           grade = " You got an F " + parseFloat(score)+"%";
+//           break;
+//         case score < 60: 
+//           grade = " You got a E " + parseFloat(score)+"%";
+//           break;
+//         case score < 70: 
+//           grade =" You got a D " + parseFloat(score)+"%";
+//           break;
+//         case score < 80: 
+//           grade = " You got a C"  + parseFloat(score)+"%";
+//           break;
+//           case score < 90: 
+//           grade = "You got a B " + parseFloat(score)+"%";
+//           break;
+//         case score <= 100:
+//           grade = "You got a A " + parseFloat(score)+"%";
+//           break;
+//       }
+//       return grade;
+// }
 
-let result = vathmos (60);
-console.log(result);
\ No newline at end of file
+// let result = vathmos (60);
+// console.log(result);
```

Comment:
Nice job, it works great! I can now calculate my grades!

Very good code refactoring that shortens and cleans your code.

Only one side note. You should leave comments on how the user can use the `gradeCalc()` function.
For example:
```
// The 1st parameter is the student's score.
// The 2nd is the grading system that is used, i.e. 20 or 100.
let gradeCalc = function(score, totalScore) { ... 
```

Also, the exercise asks for the American grading system, so by default the second argument should be 100. Nice for covering the case of different systems though, but give the user instructions on the usage.

Good work! Keep it up!